### PR TITLE
added contributing.md file

### PR DESCRIPTION
**Pull request by Sloth-Panda**
Made the contributing file and made the required changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+<p align = "center">
+
+[![built with love](https://forthebadge.com/images/badges/built-with-love.svg)]()
+
+</p>
+
+---
+
+Check out my Github profile [smaranjitghose.](https://github.com/smaranjitghose)
```
Comment (smaranjitghose) [inline CONTRIBUTING.md]: This is not necessary 😅

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Please make it more elaborate and interactive like [this](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/CONTRIBUTING.md)

**Comment by smaranjitghose** [comment]
@Sloth-Panda you are in SWOC only or DWOC as well?

**Comment by Sloth-Panda** [comment]
Sir, I am in both actually

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+**12.** To create a pull request, click on `compare and pull requests`. Please ensure you compare your feature branch to the desired branch of the repo you are suppose to make a PR to.
+
+<img src="./assets/Readme_Assets/ComparePR.png" width=600>
+
+**13.** Then add an appropriate title and description to your pull request that explains your changes and efforts done.
+
+**14.** Click on `Create Pull Request`.
+
+<img src="./assets/Readme_Assets/CreatePR.png" width=600>
```
Comment (smaranjitghose) [inline CONTRIBUTING.md:89]: These images are not there in the said path. Please add them

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+**1.** Fork [this](https://github.com/smaranjitghose/img_ai_app_boilerplate) repository.
+
+**2.** Clone your forked copy of the project.
+
+```
+git clone --depth 1 https://github.com/<your_user_name>/img_ai_app_boilerplate.git
+```
+
+<img src="assets/images/clone.png" width="500">
```
Comment (smaranjitghose) [inline CONTRIBUTING.md:27]: Make the path as `assets/Readme_Assets/`

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Pull request merged**